libexec/rtld-elf/mips/reloc.c

```diff
@@ -849,6 +875,20 @@ reloc_non_plt(Obj_Entry *obj, Obj_Entry *obj_rtld, int flags,
 				return (-1);
 			break;
 		case R_TYPE(CHERI_CAPABILITY):
+			/*
+			 * MIPS's use of Elf_Rel rather than Elf_Rela, combined
+			 * with being big-endian, makes this odd. We store the
+			 * added in the first 8 bytes, depsite that not being
```

Comment:
```suggestion
			 * addend in the first 8 bytes, depsite that not being
```

Reply:
Fixed, along with another spelling mistake in the same line, and some weird grammar in the paragraph after.